frontend/layouts/common/menus.css

```diff
@@ -19,7 +19,10 @@
     color: var(--color-golden-purple) !important;
 }
 
-.ant-menu-item-selected,
+.ant-menu-item-selected {
+	color: var(--color-golden-purple) !important;
```

Comment:
just wondering if `!important` can be avoided. I see it is used in previous code as well.